docs/executor-curl.md

```diff
@@ -45,14 +49,14 @@ Test can be run using `kubectl testkube scripts start curl-test` which gives the
 
 Script queued for execution
 
-Use following command to get script execution details:
+Use the following command to get script execution details:
 $ kubectl testkube scripts execution 613a2d7056499e6e3d5b9c3e
 
 or watch script execution until complete:
 $ kubectl testkube scripts watch 613a2d7056499e6e3d5b9c3e
 ```
 
-As in the output is stated results can be checked using `kubectl testkube scripts execution 613a2d7056499e6e3d5b9c3e` where the id of the execution is unique for each execution, make sure that the right id is used. Output of that should look something like:
+As in the output, results can be checked using `kubectl testkube scripts execution 613a2d7056499e6e3d5b9c3e`, where the id is unique for each execution. Make sure that the correct id is used. The output should look something like:
```

Comment:
Instead of ~~As in the output~~, we could say something like "As seen above" or "As you saw in the output"...